<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test PLC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="checkBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>78</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BAND_CENELEC_A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>78</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BAND_FCC</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>78</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BAND_ARIB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency Band</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CONF_CARRIER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> FIRST_CARRIER</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LAST_CARRIER</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NUM_CARRIERS</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>24</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Modulation types</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>44</y>
      <width>78</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TYPE_BPSK</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>64</y>
      <width>78</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TYPE_QPSK</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TYPE_BPSK_ROBO</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 170, 50);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num_Bytes</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Recieved Msg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num_Bytes</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num_Tx_Msg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num_Rx_Msg</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Num_lost_Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Send configuration</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Send file</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 255);
selection-background-color: rgb(0, 0, 255);
selection-color: rgb(0, 255, 0);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time(µs)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>64</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MOD_SCHEME_COHERENT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>24</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Modulation_Sheme</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>44</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MOD_SCHEME_DIFFERENTIAL</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Tone_map</string>
    </property>
    <widget class="QCheckBox" name="checkBox_12">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>T_map_0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_13">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>T_map_1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_14">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>T_map_2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_15">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>T_map_3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_16">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>T_map_4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_17">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>T_map_5</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>521</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>290</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop/Start</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>78</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TYPE_8PSK</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Latency(µs)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <zorder>checkBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>checkBox_3</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>textEdit</zorder>
   <zorder>label_4</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>label_5</zorder>
   <zorder>textEdit_3</zorder>
   <zorder>label_6</zorder>
   <zorder>checkBox_4</zorder>
   <zorder>checkBox_5</zorder>
   <zorder>checkBox_6</zorder>
   <zorder>textEdit_4</zorder>
   <zorder>label_7</zorder>
   <zorder>textBrowser</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>lcdNumber_2</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>lcdNumber_3</zorder>
   <zorder>lcdNumber_4</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>lcdNumber_5</zorder>
   <zorder>label_12</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>textEdit_5</zorder>
   <zorder>label_13</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>lcdNumber_6</zorder>
   <zorder>label_14</zorder>
   <zorder>checkBox_8</zorder>
   <zorder>label_15</zorder>
   <zorder>checkBox_9</zorder>
   <zorder>groupBox</zorder>
   <zorder>customPlot</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>checkBox_7</zorder>
   <zorder>lcdNumber_7</zorder>
   <zorder>label_16</zorder>
   <zorder>pushButton_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionClear"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
